style.css

```diff
@@ -2,7 +2,7 @@
 /*
 Theme Name: Hale
 Text Domain: hale
-Version: 1.0.3
+Version: 1.0.2
```

Comment:
This should be 1.0.3?

Reply:
Thanks actually, it needs to be one higher, updated.